operators/operating-pinot/consistent-push.md

```diff
@@ -0,0 +1,72 @@
+# Consistent Push and Rollback
+
+{% hint style="info" %}
+**Original design doc**: [https://docs.google.com/document/d/1PUy4wSUPFyEWEW3a88Mipdug3cPj4EpV__lx-BVUTYk/edit?usp=sharing](https://docs.google.com/document/d/1PUy4wSUPFyEWEW3a88Mipdug3cPj4EpV__lx-BVUTYk/edit?usp=sharing)
+
+**Issue**: [https://github.com/apache/pinot/issues/7813](https://github.com/apache/pinot/issues/7813)
+{% endhint %}
+
+### Motivation
+
+#### Data Consistency
+
+Pinot supports atomic update on segment level, which means that when data consisting of multiple segments are pushed to a table, as segments are replaced one at a time, queries to the broker during this upload phase may produce inconsistent result due to interleaving of old and new data. 
+
+#### Data Rollback
```

Comment:
Do we want to add the instructions on how to trigger the data rollback?

Reply:
Added instruction under "How to trigger Data Rollback" header, please take a look.